package.json

```diff
@@ -21,82 +21,82 @@
     "@babel/plugin-transform-regenerator": "^7.8.7",
     "@babel/plugin-transform-shorthand-properties": "^7.8.3",
     "@babel/polyfill": "^7.8.7",
-    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-env": "^7.9.5",
     "@babel/preset-react": "^7.9.4",
     "@babel/preset-typescript": "^7.9.0",
     "@centreon/frontend-core": "centreon/frontend-core",
-    "@svgr/webpack": "^5.3.0",
-    "@testing-library/jest-dom": "^5.3.0",
-    "@testing-library/react": "^10.0.1",
-    "@testing-library/user-event": "^10.0.0",
-    "@types/ramda": "^0.27.0",
-    "@types/react-router": "^5.1.4",
+    "@svgr/webpack": "^5.3.1",
+    "@testing-library/jest-dom": "^5.5.0",
+    "@testing-library/react": "^10.0.3",
+    "@testing-library/user-event": "^10.1.0",
+    "@types/ramda": "^0.27.4",
+    "@types/react-router": "^5.1.7",
     "axios-mock-adapter": "^1.18.1",
-    "babel-jest": "^25.2.3",
+    "babel-jest": "^25.4.0",
     "babel-loader": "^8.1.0",
     "babel-merge": "^3.0.0",
     "cache-loader": "^4.1.0",
     "clean-webpack-plugin": "^3.0.0",
     "clipboardy": "^2.3.0",
-    "css-loader": "^3.4.2",
+    "css-loader": "^3.5.3",
     "eslint-config-airbnb": "^18.1.0",
-    "eslint-config-prettier": "^6.10.1",
+    "eslint-config-prettier": "^6.11.0",
     "eslint-import-resolver-alias": "1.1.2",
     "eslint-plugin-babel": "^5.3.0",
     "eslint-plugin-jest": "^23.8.2",
     "eslint-plugin-prefer-arrow-functions": "^3.0.1",
-    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-prettier": "^3.1.3",
     "file-loader": "^6.0.0",
-    "fork-ts-checker-webpack-plugin": "^4.1.2",
-    "html-webpack-plugin": "^4.0.2",
-    "jest": "^25.2.3",
+    "fork-ts-checker-webpack-plugin": "^4.1.3",
+    "html-webpack-plugin": "^4.2.0",
+    "jest": "^25.4.0",
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "jest-junit": "^10.0.0",
     "lodash": "^4.17.15",
     "mini-css-extract-plugin": "^0.9.0",
-    "node-sass": "^4.13.1",
+    "node-sass": "^4.14.0",
     "postcss-loader": "^3.0.0",
     "postcss-safe-parser": "^4.0.2",
-    "prettier": "^2.0.2",
+    "prettier": "^2.0.5",
     "react-mock-router": "^1.0.15",
     "redux-mock-store": "^1.5.4",
     "resolve-url-loader": "^3.1.1",
     "sass-loader": "^8.0.2",
-    "terser-webpack-plugin": "^2.3.5",
+    "terser-webpack-plugin": "^2.3.6",
     "thread-loader": "^2.1.3",
-    "ts-loader": "^6.2.2",
+    "ts-loader": "^7.0.1",
     "typescript": "^3.8.3",
-    "url-loader": "^4.0.0",
-    "webpack": "^4.42.1",
-    "webpack-bundle-analyzer": "^3.6.1",
+    "url-loader": "^4.1.0",
+    "webpack": "^4.43.0",
+    "webpack-bundle-analyzer": "^3.7.0",
     "webpack-cli": "^3.3.11",
     "webpack-livereload-plugin": "^2.3.0",
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
     "@centreon/ui": "github:centreon/centreon-ui",
-    "@date-io/moment": "1.3.13",
-    "@material-ui/core": "^4.9.7",
+    "@date-io/moment": "2.6.0",
```

Comment:
I know this is a draft, but avoid to upgrade this lib until material-ui pickers v4 is not released